conf/defaults.yaml

```diff
@@ -259,6 +259,10 @@ topology.disruptor.batch.size: 100
 topology.disruptor.batch.timeout.millis: 1
 topology.disable.loadaware: false
 topology.state.checkpoint.interval.ms: 1000
+topology.metrics.aggregate.per.worker: false
```

Comment:
FYI, I tried to port JStorm metrics to 1.x some months ago, but it touches many paths of codebase so we would end up having hard time to do the work if port is not finished.